docs/resources/namespace_quota_policy.md

```diff
@@ -0,0 +1,743 @@
+---
+Title: "Quota Policy Resource"
+Description: |-
+    Creating the Tanzu Kubernetes quota policy resource.
+---
+
+# Quota Policy
```

Comment:
I believe all the title statements should also be renamed to namespace quota policy ?

Reply:
Done